apis/core-api/src/api/routes/v1/fanClubs/fanClubs/ControllerAdapter.ts

```diff
@@ -77,10 +77,10 @@ class ControllerAdapter {
 
   static async getById(ctx: Context): Promise<void> {
     /** VALIDATIONS, PARAMETERS */
-    const { id } = ctx.request.params;
```

Comment:
why?

Reply:
I changed the parameter name on fan club routes from `id` to `fanClubId` and forgot to update in ControllerAdapter too.

Reply:
Why not use id instead of fanClubId ?

Reply:
in fan club membership routes I'm using `:id` as membership id, so I've changes fan club `:id` with `:fanClubId`